.github/workflows/e2e.yaml

```diff
@@ -1,20 +1,32 @@
 name: e2e
 on:
   pull_request:
-    paths-ignore:
-      - '**/*.md'
   push:
     branches:
       - 'release-*'
       - 'master'
       - 'main'
     tags:
       - 'v*'
-    paths-ignore:
-      - '**/*.md'
 jobs:
+  check-only-doc-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      # Check if there are only changes to .md files
+      - name: Check for only doc file changes
+        id: doc_changes
+        run: |
+          if git status --porcelain | grep -v '\.md$'; then
```

Comment:
and if we have .md and other things together?

Reply:
so this command checks anything other than `.md` so only extra files will show in command hence `only_doc_changes` false for `if` condition. So even if `.md` and extra files are together it won't matter IIUC since we only want to make sure if anything extra is changed.

Reply:
But I am reviewing this again, will update back